diff --git a/out/codeSign/macCodeSign.js b/out/codeSign/macCodeSign.js
index 843a507ed3814f06c6ca99494ef1cf7bfb22067e..b2b27b474478d125d816b3f8e19498050ad602e6 100644
--- a/out/codeSign/macCodeSign.js
+++ b/out/codeSign/macCodeSign.js
@@ -157,15 +157,15 @@ async function createKeychain({ tmpDir, cscLink, cscKeyPassword, cscILink, cscIK
     if (cscIKeyPassword != null) {
         cscPasswords.push(cscIKeyPassword);
     }
-    return await importCerts(keychainFile, certPaths, cscPasswords);
+    return await importCerts(keychainFile, certPaths, cscPasswords, keychainPassword);
 }
-async function importCerts(keychainFile, paths, keyPasswords) {
+async function importCerts(keychainFile, paths, keyPasswords, keychainPassword) {
     for (let i = 0; i < paths.length; i++) {
         const password = keyPasswords[i];
         await (0, builder_util_1.exec)("/usr/bin/security", ["import", paths[i], "-k", keychainFile, "-T", "/usr/bin/codesign", "-T", "/usr/bin/productbuild", "-P", password]);
         // https://stackoverflow.com/questions/39868578/security-codesign-in-sierra-keychain-ignores-access-control-settings-and-ui-p
         // https://github.com/electron-userland/electron-packager/issues/701#issuecomment-322315996
-        await (0, builder_util_1.exec)("/usr/bin/security", ["set-key-partition-list", "-S", "apple-tool:,apple:", "-s", "-k", password, keychainFile]);
+        await (0, builder_util_1.exec)("/usr/bin/security", ["set-key-partition-list", "-S", "apple-tool:,apple:", "-s", "-k", keychainPassword, keychainFile]);
     }
     return {
         keychainFile,
